backend/api/serializers.py

```diff
@@ -96,7 +96,9 @@ class Meta:
 class HoldingTeamSerializer(ModelSerializer):
     class Meta:
         model = HoldingTeam
-        fields = '__all__'
+        fields = [
+            'id', 'name', 'wss', 'staff'
```

Comment:
What's hiding here?

Reply:
`order`. it's not necessary for the client, because the response is already sorted.

Reply:
Hmm, but I think budanesh khali as lotf nist :)))

Reply:
:)))))))))))))))
OK. I will add it again.